scripts/list_contributors.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Creates a list containing names of people who contributed to the project, for use
+# in release notes. The names come from the author field on the commits between
+# the current revision and the one specified as argument.
+#
+# Note that the output often requires extra manual processing to remove entries
+# that refer to the same person (diacritics vs no diacritics, name vs nickname, etc.).
+#
+# Usage:
+#    <script name>.sh <revision>
+#
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2023 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -euo pipefail
+
+script_dir=$(dirname "$0")
+# shellcheck source=scripts/common.sh
+source "${script_dir}/common.sh"
+
+(( $# == 1)) || fail "Not enough arguments. Usage: $0 <revision>."
+
+revision="$1"
+
+git shortlog --summary --email "${revision}..origin/develop" |
+    cut --field 2 |
+    sed -e 's/\s*<.*>//' |
+    sort |
+    uniq |
+    paste --serial --delimiter=, |
+    sed -e 's/,/, /g'
```

Comment:
```suggestion
git shortlog --summary "${revision}..origin/develop" |
    cut --field 2 |
    sort |
    uniq |
    paste --serial --delimiter=, |
    sed -e 's/,/, /g'
```
I think you can completely remove the email.

Reply:
Good suggestion. I didn't even notice that the e-mails show up because we explicitly request them :) Removed.

By the way, they're actually useful for manually checking if some names refer to the same person. But for that you'd run only the `shortlog` command without all the rest.